odoo/tools/query.py

```diff
@@ -272,3 +303,55 @@ def __len__(self) -> int:
 
     def __iter__(self) -> Iterator[int]:
         return iter(self.get_result_ids())
+
+    @property
+    def get(self) -> IdentifierBuilder:
```

Comment:
You could replace `env` with the `model` on the query object. That would be much simpler and give more information.

Reply:
@rco-odoo Didn't want this, wanting to keep `Query` as agnostic of the ORM as possible